component/azstorage/azauth.go

```diff
@@ -58,46 +58,12 @@ func getAzAuth(config azAuthConfig) azAuth {
 		}(config.UseHTTP),
 		config.Endpoint)
 
-	if EAccountType.BLOCK() == config.AccountType {
-		return getAzAuthBlob(config)
```

Comment:
For blob endpoint we are not returning the config here?

Reply:
Old unused code. So, removed it.
The blob endpoint calls the getAzAuthBlob() method from the track2 code.